packages/graphql-toolbox/src/contexts/auth.tsx

```diff
@@ -170,5 +133,41 @@ export function AuthProvider(props: any) {
         resolveNeo4jDesktopLoginPayload().then(processLoginPayload.bind(null, value)).catch(console.error);
     }, []);
 
-    return <AuthContext.Provider value={value }>{props.children}</AuthContext.Provider>;
+    const checkForDatabaseUpdates = async (driver: neo4j.Driver, setValue: any) => {
+        try {
+            await driver.verifyConnectivity();
+            const databases = await getDatabases(driver);
+            setValue((values) => ({ ...values, isConnected: true, databases: databases || [] }));
+        } catch (err) {
+            setValue((values) => ({ ...values, isConnected: false }));
+        }
+    };
+
+    const processLoginPayload = (value: State | undefined, loginPayloadFromDesktop: LoginPayload | null) => {
+        let loginPayload: LoginPayload | null = null;
+        if (loginPayloadFromDesktop) {
+            loginPayload = loginPayloadFromDesktop;
+            setValue((v) => ({ ...v, isNeo4jDesktop: true }));
```

Comment:
```suggestion
            setValue((values) => ({ ...values, isNeo4jDesktop: true }));
```

Just to keep consistent with other usages of setValue.